process/block/shardblock.go

```diff
@@ -432,6 +432,25 @@ func (sp *shardProcessor) requestEpochStartInfo(header data.ShardHeaderHandler,
 	return process.ErrTimeIsOut
 }
 
+func (sp *shardProcessor) requestEpochStartProofIfNeeded(hash []byte) bool {
+	if sp.epochStartTrigger.IsEpochStart() {
+		return true // no proof needed
```

Comment:
why no proof needed if epoch start? i think that only at transition not proof needed

Reply:
removed, my confusion